reference/index.yml

```diff
@@ -74,27 +74,27 @@ productDirectory:
       links:
         - url: https://www.powershellgallery.com/
           text: PowerShell Gallery
-        - url: https://docs.microsoft.com/powershell/gallery/overview
+        - url: https://docs.microsoft.com/powershell/scripting/gallery/overview
           text: What is the PowerShell Gallery
-        - url: https://docs.microsoft.com/powershell/gallery/concepts/publishing-guidelines
+        - url: https://docs.microsoft.com/powershell/scripting/gallery/concepts/publishing-guidelines
           text: Best practices
-        - url: https://docs.microsoft.com/powershell/gallery/faqs
+        - url: https://docs.microsoft.com/powershell/scripting/gallery/faqs
           text: FAQs
     # Card 4
     - title: PowerShell DSC
       imageSrc: https://docs.microsoft.com/media/common/i_config-tools.svg
       links:
-        - url: https://docs.microsoft.com/powershell/dsc/overview/overview
+        - url: https://docs.microsoft.com/powershell/scripting/dsc/overview/overview
           text: DSC Overview
-        - url: https://docs.microsoft.com/powershell/dsc/getting-started/lnxgettingstarted
+        - url: https://docs.microsoft.com/powershell/scripting/dsc/getting-started/lnxgettingstarted
           text: Get started with DSC for Linux
         - url: https://dsccommunity.org/
           text: DSC Community
     # Card 5
     - title: Developer resources
       imageSrc: https://docs.microsoft.com/media/common/i_code-edit.svg
       links:
-        - url: https://docs.microsoft.com/powershell/developer
+        - url: https://docs.microsoft.com/powershell/scripting/developer
```

Comment:
fyi: link doesn't work, but is part of the change, correct?

Reply:
fixed